nixos/modules/services/logging/journalwatch.nix

```diff
@@ -199,33 +221,34 @@ in {
 
     users.users.${user} = {
       isSystemUser = true;
-      createHome = true;
       home = dataDir;
-      # for journal access
-      group = "systemd-journal";
+      group = group;
     };
 
+    systemd.tmpfiles.rules = [
+      "d ${dataDir} 0750 ${user} ${group} - -"
```

Comment:
Please replace with [StateDirectory and StateDirectoryMode](https://www.freedesktop.org/software/systemd/man/systemd.exec.html) as this is preferred over tmpfiles.d.

Reply:
Also much more elegant, thank you! :+1: